test/test_charge_drift_models.jl

```diff
@@ -82,41 +82,109 @@ end
 end
 
 @timed_testset "Charge Trapping: ConstantLifetimeChargeTrappingModel" begin
+    # test 1: parse the lifetimes and the inactive layer geometry
+    config_dict = SolidStateDetectors.parse_config_file(SSD_examples[:TrueCoaxial])
+    simA = @test_nowarn Simulation{T}(config_dict)
+    @testset "Parse the TrueCoaxial config file" begin
+        @test simA.detector.semiconductor.charge_trapping_model isa ConstantLifetimeChargeTrappingModel{T}
+        @test simA.detector.semiconductor.charge_trapping_model.τh == T(1e-3)
+        @test simA.detector.semiconductor.charge_trapping_model.τe == T(1e-3)
+        @test simA.detector.semiconductor.charge_trapping_model.τh_inactive == T(8e-8)
+        @test simA.detector.semiconductor.charge_trapping_model.τe_inactive  == T(8e-8)
+        @test simA.detector.semiconductor.charge_trapping_model.inactive_layer_geometry.origin == CartesianPoint{T}(0.0, 0.0, 0.005)
+        r0, r1 = T.((0.009, 0.01))
+        @test simA.detector.semiconductor.charge_trapping_model.inactive_layer_geometry.r == tuple((r0, r1), (r0, r1))
+        @test simA.detector.semiconductor.charge_trapping_model.inactive_layer_geometry.hZ == T(0.005)
+    end
+
+    simA = Simulation{T}(SSD_examples[:TrueCoaxial])
+    timed_simulate!(simA, convergence_limit = 1e-5, device_array_type = device_array_type, refinement_limits = [0.2, 0.1, 0.05], verbose = false)
+    simA_inactive_layer_geometry=deepcopy(simA.detector.semiconductor.charge_trapping_model.inactive_layer_geometry)
+
+    # test2: only model the bulk volume, test the bulk signals while varying the lifetime: τ
+    pos = CartesianPoint{T}(0.005,0,0.005); Edep = 1u"eV"
     signalsum_list = []
```

Comment:
In general, if you create an `Array` to `push!` to it afterwards, consider specifying a type
```suggestion
    signalsum_list = T[]
```